docs/community-ee/community-ee-release-process.md

```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
 
-- Access to  https://github.com/ansible-community/images repo
-- Access to ghcr.io
+    - Access to  https://github.com/ansible-community/images repo
+    - Access to ghcr.io
 
+## Pre Requisite:
 
-## Build steps for Community-ee-base
+      - Join [Release Management working Matrix room](https://app.element.io/?updated=1.11.38#/room/#release-management:ansible.com) and [Execution Envs group in Forum Group](https://forum.ansible.com/g/ExecutionEnvs).
+      - Read about the Ansible Execution Environment in [here](https://forum.ansible.com/t/execution-environments-getting-started-guide-community-ee-images-availability/1341) and in [this](https://forum.ansible.com/t/execution-environments-getting-started-guide-community-ee-images-availability/1341) Forum Post.
+      - Read about  Github workflow `.github/workflows/eerelease.yml` .
+      - Show intention and book the date for the releasing of the EE.
 
-1. Go to the working directory.
+## Announcement before the release
 
-`cd images/execution-environments/community-ee-base`
+  Post this message in the channel before working on the release.
 
-2. Create a new release branch
+  ```
+  📯📯📯 I am working on the `Ansible Execution Environment Base and Minimal 2.17.1-1` Release. I will keep the room updated about the progress.📦️
+  ```
 
-`git checkout -b base-<ansible-core-version-base-ee-version>`
+## Actual Build Steps :
 
-3. Edit the `execution-environment.yml` file to the right `collection version` and `ansible-core version`. Get the correct [ansible-core](https://pypi.org/project/ansible-core/) version and [ansible-<version>.deps](https://github.com/ansible-community/ansible-build-data/blob/main/9/ansible-9.0.1.deps) and edit accordingly.
+### On the terminal
 
-`vim execution-environments/community-ee-base/execution-environment.yml`
+      - Check the ansible-core version [here](https://pypi.org/project/ansible-core/)
+      - Verify ansible collection versions for ansible.posix, ansible.utils and  ansible.windows for `deps` file of the ansible version for the related release  [ansible-build-data repo](https://github.com/ansible-community/ansible-build-data).
+      - Go to the images/execution-environments directory.
+      - Create the correct `git branch` (name it in the related version eg: 2.17.1-1)
 
+        ```
+            cd images/execution-environments
+            git checkout main
+            git pull upstream main
+            git checkout -b coreversion-eeversion
 
-4. List images in local storage
+        ```
 
-`podman images`
+      - Update the ansible-core and collection versions in the  `/images/execution-environments/community-ee-base/execution-environment.yml` file.
+      - Update the ansible-core version in the  `/images/execution-environments/community-ee-minimal/execution-environment.yml` file, commit the changes and create a PR.
 
-5. Delete the existing community-ee base and fedora images
+        ```
+            vim community-ee-base/execution-environment.yml`
```

Comment:
```suggestion
            vim community-ee-base/execution-environment.yml
```

Reply:
Maybe it's unnecessary to suggest vim here. You risk opening up the vim vs emacs can of worms. People get attached to their editors. You could just say "Open `community-ee-base/execution-environment.yml` for editing and then do: